### ISSUE TITLE
Cipher suites displays "we could not determine if the server has a preference" when server only has one supported cipher

### ISSUE THREAD

**Issue by mikkorantalainen** [ssllabs/ssllabs-scan#747]
When SSL Server Test is executed against a server that supports multiple ciphers with TLS 1.3 and TLS 1.2 but only supports one cipher with TLS 1.1 or TLS 1.0 the report page displays messages "# TLS 1.1 (we could not determine if the server has a preference)" and "# TLS 1.0 (we could not determine if the server has a preference)".

This makes the test look a bit stupid because logically there cannot be any guesses about the preference with exactly one possible choice. The code probably needs a check for the count of the possible ciphers before trying to decide if server-preferred order can be figured out.

Example:
![ssllabs-one-cipher-suite-message](https://user-images.githubusercontent.com/259482/65404629-005db780-dde2-11e9-9571-ee87ed32d984.png)

**Comment by Rachaelgomer** [comment]
Agreed! Hitting the same error, but the 1 protocol set is TLS 1.2 and it should be returning 3 cipher suites, and is instead saying it could not determine preference.
